ext/libdatadog_api/extconf.rb

```diff
@@ -67,7 +67,7 @@ def skip_building_extension!(reason)
 
 if ENV['DDTRACE_DEBUG'] == 'true'
   $defs << '-DDD_DEBUG'
-  CONFIG['optflags'] = '-O0'
```

Comment:
Is `O0` not necessary to get the debug information we need?

Reply:
I've used "-O1" with gdb and didn't see any issue/missing info, so I think it's fine, and it's easy to tweak locally -- I just wanted to avoid a default config that was broken (at least on my Ubuntu/gcc version?).